objcgen/driver.cs

```diff
@@ -59,6 +59,7 @@ public static int Main2 (params string [] args)
 			var os = new OptionSet {
 				{ "c|compile", "Compiles the generated output", v => embedder.CompileCode = true },
 				{ "e|extension", "Compiles the generated output as extension safe api", v => embedder.Extension = true },
+				{ "nobitcode", "Compiles the generated output without bitcode enabled", v => embedder.NoBitcode = true },
```

Comment:
Based on past experience using a negative is not the best pattern.
It's better to have something like `bitcode=[true|false|default]`